Gemfile

```diff
@@ -1,6 +1,6 @@
 source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
-if puppetversion = ENV['PUPPET_GEM_VERSION']
+if puppetversion = ENV['PUPPET_GEM_VERSION'] || "~> 5.x"
```

Comment:
The Gemfile errors out if `PUPPET_GEM_VERSION` is not set because `puppetversion` ends up being nil.  Without this change `PUPPET_GEM_VERSION` would have to be defined to the acceptance tests even though it has not bearing on the version of Puppet used by the tests.

```
$ bundle install
The latest bundler is 1.16.1, but you are currently running 1.15.4.
To update, run `gem install bundler`

[!] There was an error parsing `Gemfile`: undefined method `<' for nil:NilClass. Bundler cannot continue.

 #  from /Users/tdockendorf/git/sensu-puppet/Gemfile:37
 #  -------------------------------------------
 #    gem 'puppet-lint-version_comparison-check',             :require => false
 >    if puppetversion < '5.0'
 #      gem 'semantic_puppet', :require => false
 #  -------------------------------------------
```